app/models/resource.rb

```diff
@@ -34,18 +34,28 @@ def create_new_draft(language_id)
     Translation.create!(resource: self, language: language)
   end
 
-  # Returns the latest translation for each language.  In more detailed terms, the inner query returns the highest
-  # version for the resource/language combination. The outer query then finds the translation matching that version.
   def latest_translations
-    Translation.find_by_sql("select T.* from translations T inner join (select language_id, resource_id, max(version) as
-                            max_version from translations where is_published = true and resource_id = #{id} group
-                            by language_id, resource_id) as max_table on T.version = max_table.max_version and
-                            T.language_id = max_table.language_id and T.resource_id = max_table.resource_id")
+    latest('is_published = true and')
```

Comment:
This feels a little dirty. I think what you're trying to accomplish in the `#latest` method below can be accomplished more cleanly by putting that subquery into the `#from` method: http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-from